crates/bevy_core_pipeline/src/core_3d/main_transmissive_pass_3d_node.rs

```diff
@@ -52,72 +54,98 @@ impl ViewNode for MainTransmissivePass3dNode {
             return Ok(());
         };
 
-        let physical_target_size = camera.physical_target_size.unwrap();
-
-        let render_pass_descriptor = RenderPassDescriptor {
-            label: Some("main_transmissive_pass_3d"),
-            color_attachments: &[Some(target.get_color_attachment())],
-            depth_stencil_attachment: Some(depth.get_attachment(StoreOp::Store)),
-            timestamp_writes: None,
-            occlusion_query_set: None,
-        };
-
-        // Run the transmissive pass, sorted back-to-front
-        // NOTE: Scoped to drop the mutable borrow of render_context
-        #[cfg(feature = "trace")]
-        let _main_transmissive_pass_3d_span = info_span!("main_transmissive_pass_3d").entered();
-
         if !transmissive_phase.items.is_empty() {
```

Comment:
This should be an early return to reduce nesting. Nothing else happens if this check is false